lib/rabbitmq/cli/ctl/commands/add_vhost_command.ex

```diff
@@ -47,4 +58,14 @@ defmodule RabbitMQ.CLI.Ctl.Commands.AddVhostCommand do
   def description(), do: "Creates a virtual host"
 
   def banner([vhost], _), do: "Adding vhost \"#{vhost}\" ..."
+
+  #
+  # Implementation
+  #
+
+  def parse_tags(tags) do
+    String.split(tags, ",", trim: true)
+    |> Enum.map(&String.trim/1)
+    |> Enum.map(&String.to_atom/1)
```

Comment:
I suppose we can use the existing function in rabbitmq-server

Reply:
Also same comment as the one attached the other copy of `parse_tags()`: do we really need atoms here?

Reply:
We agreed to use atoms in rabbitmq/rabbitmq-server#2080. CLI tools try to depend as little as possible on the server modules and when it comes to working with strings, I'd take Elixir over Erlang any day.